vendor/modules.txt

```diff
@@ -10,6 +10,8 @@ github.com/PuerkitoBio/urlesc
 github.com/asaskevich/govalidator
 # github.com/beorn7/perks v1.0.1
 github.com/beorn7/perks/quantile
+# github.com/caarlos0/env v3.5.0+incompatible
```

Comment:
it's not added anywhere

Reply:
This dependency is declared here https://github.com/percona/percona-xtradb-cluster-operator/blob/main/cmd/pitr/main.go#L12 and it wasn't in go.mod file, so I added it.
The go.sum file changed automatically when you add or remove something from go.mod file.